src/BaseControllerV2.test.ts

```diff
@@ -511,4 +571,149 @@ describe('getPersistentState', () => {
 
     expect(persistentState).toEqual({ count: 1 });
   });
+
+  describe('inter-controller communication', () => {
```

Comment:
nit: It'd be nice if we had some breathing space between the type declarations in here.

Reply:
Done in d25bb8e692d21cdb7ccd00492c1e510ec5942c5e

I left the type declarations together, because it still felt right to group them together. But now there are newlines around them, and around the metadata object.